app/Http/Controllers/Api/V1/MicroSoftTeamController.php

```diff
@@ -112,18 +114,15 @@ public function getAccessToken(Request $request)
     /**
 	 * get access_token using code
 	 *
-	 * get access_token using code.
-	 *
-     * @urlParam gid string User id of current logged in user
-     * @bodyParam code string The stringified of the GAPI authorization token JSON object
+     * @bodyParam request contains classId, assignmentId, submissionId
```

Comment:
"c" of "contains" should be uppercase.
same is the case with "g" for "get access_token using code".

Also "* @param Request $request" should instead be "* @param Request GetTokenViaCode".